### PR TITLE
Legacy test setup and login/logout tests

### PR DESCRIPTION
**Pull request by Dantemss**
You will probably have to `rm -rf .tox` because I couldn't figure out a way to make tox update its virtualenv with the stuff I added to requirements.txt

This PR adds:
- Legacy test setup
- Legacy login and logout tests

To run the login/logout tests, you need to add LEGACY_USERNAME and LEGACY_PASSWORD to a .env file at the root of the repo and then:

`tox -- -m legacy --runslow`

### CODE REVIEW

File: pages/legacy/base.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+class Region(pypom.Region):
+
+    # Backported from PyPOM 2.0.0. Remove when upgrading to PyPOM 2.0.0+
+    @property
+    def loaded(self):
+        return True
+
+    # Backported from PyPOM 2.0.0. Remove when upgrading to PyPOM 2.0.0+
```
Comment (m1yag1) [inline pages/legacy/base.py]: I didn't realize our PyPom was so old. Why not upgrade it?

Comment (m1yag1) [reply pages/legacy/base.py]: I went ahead and upgraded us to 2.0 ;).

File: pages/legacy/my_account.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from selenium.webdriver.common.by import By
+
+from pages.legacy.base import Region
+from pages.legacy.my_dashboard import MyDashboard
+from pages.legacy.login_form import LoginForm
+
+
+class MyAccount(Region):
+
+    _LOGIN_FORM_LOCATOR_ = (By.CSS_SELECTOR, 'form[action$="login_form"]')
```
[the remark this answers is not in the document]
Comment (Dantemss) [reply pages/legacy/my_account.py]: OK I will convert to lowercase and remove the trailing underscore

File: pages/legacy/base.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    @property
+    def can_login(self):
+        return False
+
+    @property
+    def loaded(self):
+        return super().loaded and self.has_username and self.can_logout
+
+
+class Region(pypom.Region):
```
[the remark this answers is not in the document]
Comment (Dantemss) [reply pages/legacy/base.py]: Hm I intended this to be the base class for all regions. Do you want this separate from the base Page?

The cyclic import is caused by the Page instantiating a My Account Region, which instantiates other pages. I see the way you avoided this was by having the import in the method code, but I think this should be the same as having the import at the end of the file?

File: pages/legacy/my_account.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    _USERNAME_FIELD_LOCATOR_ = (By.ID, '__ac_name')
+    _PASSWORD_FIELD_LOCATOR_ = (By.ID, '__ac_password')
+
+    _USERNAME_SPAN_LOCATOR_ = (By.ID, 'user-name')
+    _LOGOUT_FORM_LOCATOR_ = (By.CSS_SELECTOR, 'form[action$="logout"]')
+
+    @property
+    def can_login(self):
+        return self.is_element_displayed(*self._LOGIN_FORM_LOCATOR_) and \
```
[the remark this answers is not in the document]
Comment (Dantemss) [reply pages/legacy/my_account.py]: Hm ok, I'll change. Flake8 did not complain about this one.

File: pages/legacy/base.py (hunk 1 above)
Comment (Dantemss) [reply pages/legacy/base.py]: Ah that might mean I no longer need the base Region.

File: pages/legacy/base.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (m1yag1) [reply pages/legacy/base.py]: It's better form to import from within a method than it is to do the import at the bottom of the file.

File: pages/legacy/base.py (hunk 1 above)
Comment (Dantemss) [reply pages/legacy/base.py]: I left the base Region in, but it is currently empty.

File: pages/legacy/base.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dantemss) [reply pages/legacy/base.py]: OK I will put the import in the method then

**Pull request merged**